analytics_data_api/v0/serializers.py

```diff
@@ -332,6 +332,14 @@ class LearnerSerializer(serializers.Serializer, DefaultIfNoneMixin):
     enrollment_date = serializers.DateField(source='enrollment_date', format=settings.DATE_FORMAT)
     cohort = serializers.CharField(source='cohort')
 
+    def transform_segments(self, _obj, value):
+        # returns null instead of empty strings
+        return value or None
+
+    def transform_cohort(self, _obj, value):
+        # returns null instead of empty strings
+        return value or None
```

Comment:
Looks like the empty string cohort does not actually happen. At least, the UI blocks you from entering a cohort with an empty name. Sorry for the confusion!

Given that, I like how this makes things work. `null` means the student belongs to no cohort, while any string means the student belongs to a cohort with that string as the name.